### ISSUE TITLE
Bump Sendgrid http client library to use latest apache http client 5

### ISSUE THREAD

**Issue by ahrooran-r** [sendgrid/java-http-client#147]
This latest version of sendgrid does not support apache http client 5.
I saw another issue where you are trying to use JDK specific http libraries.
However, its been in stale since 2016 and last comment during 2022 also has no useful updates.

Could you please bump the library to use Http Client 5 at least?
https://mvnrepository.com/artifact/org.apache.httpcomponents.client5/httpclient5

**Comment by jkosternl** [comment]
Yes, for our project https://github.com/frankframework/frankframework we are using SendGrid as well, which uses this `java-http-client` which depends on the old `org.apache.httpcomponents:httpclient:4.*`.

**Comment by ryantomlinson95** [comment]
I raised a PR to bump the library to use httpclient5, since we use SendGrid in our project that we're upgrading to httpclient5.

**Comment by jkosternl** [comment]
Great job @ryantomlinson95 👍🏻  
I hope the maintainers can proceed with your changes and merge it and release a new version.
